sklearn/linear_model/tests/test_logistic.py

```diff
@@ -1419,15 +1419,17 @@ def test_n_iter(solver):
     assert clf.n_iter_.shape == (1, n_cv_fold, n_Cs)
 
     # multinomial case
-    n_classes = 1
     if solver in ("liblinear", "sag", "saga"):
+        # Those solvers only support one-vs-rest multiclass classification.
```

Comment:
It's true for liblinear but sag and saga do support multinomial. You can try to remove them for the list maybe ?

Reply:
Indeed sag and saga do work.